Bootstrap.bat

```diff
@@ -0,0 +1,101 @@
+@ECHO OFF
+SETLOCAL
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+SET vsversion=%1
+IF "%vsversion%" == "" (
+	SET vsversion=vs2015
```

Comment:
Perhaps this should default to vs2019? Or nothing at all?

Reply:
I added a default so that the bat can be executed through the file explorer.
I set vs2015 as it's a sort of default in my head. I can set vs2019 if you find it more appropriate.

Reply:
My thoughts behind this is that anything that isn't latest would just appear as arbitrary. Having this prompt for a version would probably be the best option for those running through file explorer.